pkg/kubelet/eviction/helpers_others.go

```diff
@@ -0,0 +1,37 @@
+//go:build !windows
```

Comment:
This file can safely be called helpers_unix.go or helpers_linux.go

Reply:
We do build for `darwin` now which I believe should have the same implementation as linux.
I think _others and _windows is cleaner but let me know and I can make a _linux, _windows, _darwin (or _others)

Reply:
This is probably ok for now. Thanks!